ddtrace/tracer/option.go

```diff
@@ -102,6 +102,30 @@ func defaults(c *config) {
 			log.Warn("unable to look up hostname: %v", err)
 		}
 	}
+
+	if v := os.Getenv("DD_TAGS"); v != "" {
+		for _, tag := range strings.Split(v, ",") {
+			if tag == "" {
+				continue
+			}
+
```

Comment:
Can you please remove this line? The curly braces already provide enough spacing.